test/libyul/ewasmTranslationTests/arithmetic_sdiv.yul

```diff
@@ -0,0 +1,33 @@
+{
+  sstore(0, sdiv(0, 1))
```

Comment:
This is missing in the polyfill:
```
std::exception::what: :2:13: Error: Function not found.
  sstore(0, sdiv(0, 1))
            ^--^
```

Please add a dummy one with `unreachable`.

Reply:
Done